.deployment.config.json

```diff
@@ -190,7 +190,8 @@
     "no_endpoint": true
   },
   "package_rollout": {
-    "only_consider_changesets_after": "b244fe702d8e96d016a52715e92c8131acfde3ba"
+    "only_consider_changesets_after": "b244fe702d8e96d016a52715e92c8131acfde3ba",
+    "disabled": $[STOP_AT_DEV]
```

Comment:
The quotation marks are missing here, and it's on purpose. This will be replaced by the variable directly.